Demo/src/main/res/xml/settings.xml

```diff
@@ -91,6 +91,12 @@
     <PreferenceCategory
         android:title="@string/paypal">
 
+        <CheckBoxPreference
```

Comment:
What are your thoughts on adding a setting where we have a dropdown of tokenization keys? We could even have an option to input a tokenization key in the demo app settings?

I'm thinking that the "new PayPal checkout experience" will eventually be rolled out and become the normal flow and we'll need to remove this preference when that happens.

Reply:
Yes I think that's a great idea - can you make a ticket for that?